### DISCUSSION TITLE
Adjusting Injection Rate in GEOPHIRES-X Simulation: Missing Parameter Issue

### DISCUSSION THREAD

**Discussion by huhanyu** [NatLabRockies/GEOPHIRES-X#88] [Q&A] [answered]
While using GEOPHIRES-X to simulate EGS, I encountered an issue regarding the adjustment of the Injection rate (kg/s) to optimize its performance. However, in the parameter settings, I could not find the option for Injection rate, but only Injectivity Index. As I understand it, Injectivity Index is a reservoir property obtained through experiments, rather than a variable that can be adjusted. Could you please advise me on how to approach optimizing the Injection rate in this case? I look forward to your response.

**Comment by kfbeckers** [comment, marked as answer]
The user only specifies the production well flow rate and the water loss rate in the reservoir.
The injection flow rate is internally calculated as production flow rate * (1 + water loss rate).
Indeed, the Injectivity Index is a separate parameter used when calculating the injection wellbore pressure drop.

**Reply by huhanyu** [reply]
Thanks for your reply, I get it.
